content/en/04-tutorials/03-integrations/03-otel.md

```diff
@@ -161,6 +182,94 @@ data=[
 ]
 ```
 
-
 We currently support a limited set of OpenTelemetry signal types: `metric.Gauge`, `metric.Summary` and `trace.Span`. We also support a subset of the available fields for each instrument. You can see the full set of features [in our api documentation.](/reference/pxl/otel-export) If you want support for other fields, please [open an issue](https://github.com/pixie-io/pixie).
 
+## Setup the Plugin System
+
+Now that we have a PxL script that exports OTel data, let's set up the [Plugin System](/reference/plugins/plugin-system/) to run this script at regularly scheduled intervals.
+
+1. Navigate to the `Plugin` tab on the `Admin` page (`/admin/plugins`).
+
+2. Click the toggle to enable the OpenTelemetry plugin.
+
+3. Expand the plugin row (with the arrow next to the toggle) and enter the export path.
+
+> If you are using the demo collector from [Step 1](/tutorials/integrations/otel/#deploy-the-otel-collector), then you'll use the same export path pictured below: `otel-collector.default.svc.cluster.local:4317`
+
+4. Click the toggle to disable "Secure connections with TLS" and press the `SAVE` button. The demo OTel collector does not support TLS.
+
+<svg title='Plugins are enabled in the Admin UI.' src='plugin/plugins_page.png'/>
+
+5. Click the database icon in the left nav bar to open the data export configuration page.
+
+<svg title='Select the data export icon.' src='plugin/data-export-icon.png'/>
+
+6. The OpenTelemetry plugin comes with several pre-configured OTel export PxL scripts (more coming soon!). Click the toggle to disable these scripts:
```

Comment:
"Click the toggle to disable these scripts for this tutorial, to ensure the data the collector receives is actually from our custom script"